File: examples/serverless/nodejs/hello-world-ts/handler.ts
'use strict';

export async function hello(event) {
  return {
    statusCode: 200,
    body: JSON.stringify(
      {
        message: '!Go Serverless v1.0! Your function executed successfully!',
        input: event,
      },
      null,
      2
    ),
  };

  // Use this code if you don't use the http event with the LAMBDA-PROXY integration
  // return { message: 'Go Serverless v1.0! Your function executed successfully!', event };
};

File: examples/serverless/python/hello-world-debug/handler.py
import json
import os

if "IS_LOCAL" in os.environ and os.environ["IS_LOCAL"]:
    import ptvsd
    ptvsd.enable_attach(address=('0.0.0.0', os.environ["MERLOC_DOCKER_DEBUG_PORT"]), redirect_output=True)
    ptvsd.wait_for_attach()

def hello(event, context):
    body = {
        "message": "Go Serverless v3.0! Your function executed successfully!",
        "input": event,
    }

    return {"statusCode": 200, "body": json.dumps(body)}
